### PR TITLE
7.post comment

### PR DESCRIPTION
**Pull request by aiidanm**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -206,8 +207,79 @@ describe("400 bad request incorrect id type", () => {
...
+  it("should respond with the posted comment", () => {
+    const testComment = {
+      username: "bainesface",
+      body: "i disagree with this review",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(testComment)
+      .expect(201)
+      .then(({ body }) => {
```
Comment (pippaamy) [inline __tests__/app.test.js:226]: add 201 test that ignores unnecessary properties

File: __tests__/app.test.js [hunk 2]
```diff
@@ -206,8 +207,79 @@ describe("400 bad request incorrect id type", () => {
...
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({ username: "Aidan", body: "i hate this review" })
+        .expect(404);
+    });
+    it("400: POST: should return a 400 if the body is not a valid string", () => {
+      return request(app)
+        .post("/api/reviews/3/comments")
+        .send({ username: "bainesface", body: 0 })
+        .expect(400)
```
Comment (pippaamy) [inline __tests__/app.test.js:280]: need 404 test to check if invalid id

File: __tests__/app.test.js [hunk 3]
```diff
@@ -206,8 +207,79 @@ describe("400 bad request incorrect id type", () => {
...
+        .then((response) => {
+          const msg = response.body.msg;
+          expect(msg).toBe("body needs to be a string");
+        });
+    });
+    it("404: Post: should return a 404 error if the passed object doesnt contain the correct keys", () => {
```
Comment (pippaamy) [inline __tests__/app.test.js]: this test should be a 400 :)

**Review by pippaamy — CHANGES_REQUESTED** [review]
Some changes to make in your tests! can you change these then do a new pr please :)nice work!

**Review by Keb84 — APPROVED** [review]
Looks good. Feel free to Merge 🙂

**Pull request merged**